spec/rspec/core/configuration_spec.rb

```diff
@@ -125,6 +125,12 @@ module RSpec::Core
           expect(config).not_to have_received(:warn)
         end
       end
+
+      it 'changes output stream of already created formatters' do
+        io = double 'output io'
+        expect(config.formatter_loader).to receive(:change_output_stream).with(io)
+        config.output_stream = io
```

Comment:
We use a `StringIO` to represent IO objects in this spec, would be nice to match that convention here, it has the advantage of being a ready made "double" for IO.